rest-api/dao/metrics_cache_dao.py

```diff
@@ -985,6 +985,49 @@ def to_client_json(self, result_set):
       client_json.append(new_item)
     return client_json
 
+  def to_public_metrics_client_json(self, result_set):
+    client_json = []
+    for record in result_set:
+      new_item = {
+        'date': record.date.isoformat(),
+        'metrics': {
+          'completed': {
+            'Registered': record.registered,
+            'Consent_Enrollment': record.consentEnrollment,
```

Comment:
Is there a reason for returning a new column name instead of the model column name here? It seems like a standard that APIs return the model column name.

Reply:
Yes, I agree with you, that make things easy for us. I think the current situation is that all the JSON data are returned with a "_" format name, this may inherited from the old APIs. And now all the APIs are using by the dashboard frontend, it's not easy to change them unless we make two versions of them and ask healthPro to update to the new one.

Reply:
Ok, I guess its a design decision we are stuck with.